pages/_app.tsx

```diff
@@ -45,7 +46,6 @@ export default function Layout({
         <title>BitcoinBeach Lightning Node</title>
       </Head>
       <GraphQLProvider>
-        <Header />
```

Comment:
Removing the header has left our pay page looking very bare - is this intentional?

Reply:
Yeah. But I guess Mili can give us a new updated design